src/runtime.js

```diff
@@ -513,6 +513,36 @@ export class MultidocManager {
       amp.resources = Services.resourcesForDoc(ampdoc);
     }
 
+    /**
+     * Expose amp-bind methods
+     */
+    amp['bind'] = {
+      /**
+       * Expose amp-bind getState
+       * @param {string} name - Name of state or deep state
+       * @return {Promise<(?JsonObject|string|undefined)>} - Resolves to
+       * a copy of the value of a state
+       */
+      getState: name => {
+        return Services.bindForDocOrNull(shadowRoot).then(bind => {
+          return bind && bind.getStateCopy(name);
```

Comment:
This should probably be changed to:  
`return bind ? bind.getStateCopy(name) : Promise.resolve()`;